lib/certificate_chain_serializer.rb

```diff
@@ -3,7 +3,7 @@ class CertificateChainSerializer
 
   def self.load(chain)
     return [] unless chain
-    chain.scan(PATTERN).map do |cert|
+    chain.scan(PATTERN).map! do |cert|
```

Comment:
Is this change related to the functionality?

Reply:
No functional change